web/stories/Molecules/Button/PrimaryLarge.stories.tsx

```diff
@@ -25,42 +25,10 @@ PrimaryColor.args = {
   isLargeButton: true,
 };
 
-export const SecondaryColor = Template.bind({});
-
-SecondaryColor.args = {
```

Comment:
[dust] Here's an example of the design system not being up-to-date. We're currently using some of these colors for the primary large button on the homepage. I suggest keeping these stories and adding a comment to the design team to update the design system to include these colors as well.

<img width="794" alt="Screenshot 2023-09-28 at 4 12 58 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/fa4ca7a2-8d18-4894-8ab9-b31da7139365">